mangohud/mangohud.spec

```diff
@@ -0,0 +1,385 @@
+
+Provides:       bundled(imgui) = %{imgui_ver}
+Provides:       bundled(vulkan-headers) = %{vulkan_headers_ver}
+
+%global _description %{expand:
+A Vulkan and OpenGL overlay for monitoring FPS, temperatures, CPU/GPU load and
+more.
+
+To install GUI front-end:
+
+  # dnf install goverlay
+
+To install local visualization "mangoplot":
+
+  # dnf install %{name}-mangoplot}
+
+%description %{_description}
+
+
+%package        mangoplot
+Summary:        Local visualization "mangoplot" for %{name}
+BuildArch:      noarch
+
+Requires:       %{name} = %{version}-%{release}
+Requires:       python3-matplotlib
+Requires:       python3-numpy
+
+%description    mangoplot
+Local visualization "mangoplot" for %{name}.
+
+
+%prep
+%forgeautosetup -p1
+%setup -qn %{appname}-%{tarball_version} -D -T -a1
+%setup -qn %{appname}-%{tarball_version} -D -T -a2
+%setup -qn %{appname}-%{tarball_version} -D -T -a3
+%setup -qn %{appname}-%{tarball_version} -D -T -a4
+%setup -qn %{appname}-%{tarball_version} -D -T -a5
+%setup -qn %{appname}-%{tarball_version} -D -T -a6
+
+# imgui
+mv imgui-%{imgui_ver} subprojects/
+# Vulkan-Headers
+mv Vulkan-Headers-%{vulkan_headers_ver} subprojects/
+# implot
+mv implot-%{implot_ver} subprojects/
+
+%if %{with tests}
+# Use system cmocka instead of subproject
+# https://gitlab.archlinux.org/archlinux/packaging/packages/mangohud/-/blob/0.6.9.1-10/PKGBUILD?ref_type=tags#L32
+sed -i "s/  cmocka = subproject('cmocka')//g" meson.build
+sed -i "s/cmocka_dep = cmocka.get_variable('cmocka_dep')/cmocka_dep = dependency('cmocka')/g" meson.build
+%endif
+
+# https://github.com/flightlessmango/MangoHud/commit/dc1761e98a435aaee6a919e21f43b85cc38500ac
+sed -i "s/, '-static-libstdc++'//" \
+    src/meson.build
+
+
+%build
+%meson \
+    -Dmangoapp=true \
+    -Dmangoapp_layer=true \
+    -Dmangohudctl=true \
+    -Dinclude_doc=true \
+    -Duse_system_spdlog=enabled \
+    -Dwith_wayland=enabled \
+    -Dwith_xnvctrl=disabled \
+    %if %{with tests}
+    -Dtests=enabled \
+    %else
+    -Dtests=disabled \
+    %endif
+    %{nil}
+%meson_build
+
+
+%install
+%meson_install
+
+# ERROR: ambiguous python shebang
+sed -i "s@#!/usr/bin/env python@#!/usr/bin/python3@" \
+    %{buildroot}%{_bindir}/mangoplot
+
+
+%check
+# https://github.com/flightlessmango/MangoHud/issues/812
+# ? tag-invalid           : stock icon is not valid [io.github.flightlessmango.mangohud]
+%dnl appstream-util validate-relax --nonet %{buildroot}%{_metainfodir}/*.xml
+%if %{with tests}
+%meson_test
+%endif
+
+
+%files
+%license LICENSE
+%doc README.md presets.conf.example
+%{_bindir}/%{name}*
+%{_bindir}/mangoapp
+%{_datadir}/icons/hicolor/scalable/*/*.svg
+%{_datadir}/vulkan/implicit_layer.d/*Mango*.json
+%{_docdir}/%{name}/%{appname}.conf.example
+%{_libdir}/%{name}/
+%{_mandir}/man1/%{name}.1*
+%{_mandir}/man1/mangoapp.1*
+%{_metainfodir}/*.metainfo.xml
+
+%files mangoplot
+%{_bindir}/mangoplot
+
+
+%changelog
+* Wed Mar 10 2024 Alesh Slovak <aleshslovak@gmail.com> - 0.7.1-4
```

Comment:
I get this warning about the date when building a binary RPM:

```
$ rpmbuild -bs mangohud.spec 
warning: bogus date in %changelog: Wed Mar 10 2024 Alesh Slovak <aleshslovak@gmail.com> - 0.7.1-4
```

I think you meant to use `Apr` instead of `Mar`.

Reply:
Oops. Nice catch!